helm/default-apps-vsphere/values.schema.json

```diff
@@ -1,91 +1,355 @@
 {
-    "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$defs": {
-        "app": {
-            "additionalProperties": false,
+    "$schema": "http://json-schema.org/schema#",
```

Comment:
@marians Are you OK with this change?

Reply:
FTR I used helm schema-gen to generate the schema from the values file.

Reply:
Since this is a default-apps chart an d not a cluster-PROVIDER one, our strict rules for JSON schema do not apply.

However, it looks like somebody in the past introduces reusable schema parts (in `$defs`) and decided to use draft `2020-12`. I would not simply revert this.

Reply:
Switched back to `https://json-schema.org/draft/2020-12/schema` 👍🏻